include/ktxvulkan.h

```diff
@@ -62,6 +98,7 @@ extern "C" {
  */
 typedef struct ktxVulkanTexture
 {
+    ktxVulkanFunctions vkFuncs; /*!< Handle to the functions used to allocate the texture */
```

Comment:
Why do you have the functions both here and in `ktxVulkanDeviceInfo`? I now see why having reviewed to the end. Should we make a different structure just for the 2 functions that `ktxVulkanTexture_Destruct` uses instead of bloating every texture with the full function list?

Reply:
This is really not my domain, I just went with what was easiest at first, to get feedback. I don't know how `ktxVulkanTexture` is used in real life, if it will ever be in the inner loop and so on. Adding an additional structure would mean having to modify that if we ever add more functions, but I don't see it as a big problem.

This is really just up to you :)

Reply:
It won't ever be in the inner loop but there'll be one `ktxVulkanTexture` for every texture created and some apps create a lot of textures. So I'd prefer to only have pointers to the functions actually needed.